### ISSUE TITLE
Re-raising an exception is not properly handled (google-style only)

### ISSUE THREAD

**Issue by leandro-lucarella-frequenz** [jsh9/pydoclint#16]
```console
luca@frq-lap-019:/tmp/venv [venv] 1 ! $ pip show pydoclint 
Name: pydoclint
Version: 0.0.6
Summary: A linter to check arguments in Python docstrings
Home-page: https://github.com/jsh9/pydoclint
Author: 
Author-email: 
License: MIT
Location: /tmp/venv/lib/python3.11/site-packages
Requires: click, docstring-parser, flake8, numpydoc
Required-by: 
luca@frq-lap-019:/tmp/venv [venv] $ cat t.py 
def f() -> None:
    """This is a function.

    Raises:
        FileNotFoundError: If the file does not exist.
    """
    try:
        open("nonexistent")
    except FileNotFoundError:
        raise
luca@frq-lap-019:/tmp/venv [venv] $ pydoclint t.py 
Skipping files that match this pattern: \.git|\.tox
t.py
🎉 No violations 🎉
luca@frq-lap-019:/tmp/venv [venv] $ pydoclint --style google t.py 
Skipping files that match this pattern: \.git|\.tox
t.py
t.py
    1: DOC502: Function `f` has a "Raises" section in the docstring, but there are not "raise" statements in the body 
```

**Comment by jsh9** [comment]
Thanks!  This is a good catch.

I fixed it in https://github.com/jsh9/pydoclint/commit/b9f9f6f5e819247b02c91f8f46713aa306e3b163

This fix is included in the newest release (0.0.7).

**Comment by jsh9** [comment]
Closing this issue as 'fixed'.
